dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1125,6 +1173,77 @@
         }
       }
     },
+    "EntityComponentInformation": {
+      "description": "The object for entity component information.",
+      "type": "object",
+      "discriminator": "componentKind",
+      "properties": {
+        "componentKind": {
+          "description": "The component object kind.",
+          "type": "string",
+          "enum": [
+            "PrebuiltComponent",
+            "LearnedComponent",
+            "ListComponent"
```

Comment:
In addition to what @kristapratico said about the name to better differentiate against something as common as "ComponentKind" (for a generate model name), enum values should also be camelCase. If PascalCase is ubiquitous, I would still ask that you verify this matches what the code does. Several times in Language, the enum values in the swagger and what's actually returned/accepted haven't aligned and it's been very problematic.

Reply:
For componetKind, I feel adding "entity" is redundant. However, we can get the board feedback in this week's meeting and will follow the consensus.

For the enum casing, I'm following a similar structure in CLU [here ](https://github.com/Azure/azure-rest-api-specs/blob/main/dev/cognitiveservices/data-plane/Language/analyzeconversations.json#L1510) .. The enum is PascalCase, and I'm trying to align with existing APIs as much as possible.

Noted for the alignment with the service code. @mshaban-msft Please take a note of this. Let's make sure the service APIs respect the casing of the swagger enums.